Content.Shared/Actions/BaseActionPrototype.cs

```diff
@@ -19,35 +20,35 @@ public abstract class BaseActionPrototype : IPrototype
         /// Icon representing this action in the UI.
         /// </summary>
         [ViewVariables]
-        public SpriteSpecifier Icon { get; private set; }
+        public SpriteSpecifier Icon { get; private set; } = SpriteSpecifier.Invalid;
 
         /// <summary>
         /// For toggle actions only, icon to show when toggled on. If omitted,
         /// the action will simply be highlighted when turned on.
         /// </summary>
         [ViewVariables]
-        public SpriteSpecifier IconOn { get; private set; }
+        public SpriteSpecifier IconOn { get; private set; } = SpriteSpecifier.Invalid;
 
         /// <summary>
         /// Name to show in UI. Accepts formatting.
         /// </summary>
-        public FormattedMessage Name { get; private set; }
+        public FormattedMessage Name { get; private set; } = new();
 
         /// <summary>
         /// Description to show in UI. Accepts formatting.
         /// </summary>
-        public FormattedMessage Description { get; private set; }
+        public FormattedMessage Description { get; private set; } = new();
 
         /// <summary>
         /// Requirements message to show in UI. Accepts formatting, but generally should be avoided
         /// so the requirements message isn't too prominent in the tooltip.
         /// </summary>
-        public string Requires { get; private set; }
+        public string Requires { get; private set; } = null!;
```

Comment:
null! is suss and this should be handled.

Reply:
Replaced with `string.Empty`.